simple_salesforce/api.py

```diff
@@ -135,9 +142,23 @@ def __init__(
             'X-PrettyPrint': '1'
         }
 
+        self.bulk_headers = {
```

Comment:
Is there a reason to have these as public attributes? I know it's consistent with `self.headers`, but do you foresee them being used external to this library? If not, I think I'd prefer to see these as literals in `_call_bulk_api`.